packages/@okta/vuepress-site/docs/reference/api/zones/index.md

```diff
@@ -6,11 +6,11 @@ category: management
 
 > This API is an <ApiLifecycle access="ea" /> feature.
 
-The Okta Zones API provides operations to manage zones in your organization. Zones may be used to guide policy decisions.
+The Okta Zones API provides operations to manage zones in your organization. There are two type of zones: Policy Network Zones and Block List Network Zones. Policy Network Zones may be used to guide policy decisions, and Block List Network Zones are used to deny access from certain IP addresses or locations before policy evaluation.
```

Comment:
`...and Block List Network Zones are used to block access certain IP addresses or locations or proxy types or ASNs befor policy evaluation..`?